open_schools_platform/common/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+
+class InputFilter(admin.SimpleListFilter):
+    template = 'templates/input_filter.html'
+
+    def lookups(self, request, model_admin):
+        return ((),)
```

Comment:
wow!
what is it?)

Reply:
Yeah, it looks strange))
But we need this for showing custom filter in admin. Django demands redefined lookups method for custom filters, even if it is empty.